.travis.yml

```diff
@@ -1,16 +1,17 @@
----
-language: c
-
-before_install:
-  - sudo apt-get install gfortran
-  - sudo apt-get install ruby-full
-
+dist: trusty
+sudo: false 
+#language: fortran
+language: C
+branches:
+  only:
+  - fortran-cmake
```

Comment:
The tests should be run on all branches/PRs, so I think you can remove the `branches` section. If a branch modifies `.travis.yml`, Travis will run it only on that branch.